guide/src/python_from_rust.md

```diff
@@ -236,6 +236,75 @@ def leaky_relu(x, slope=0.01):
 # }
 ```
 
+### Want to embed Python in Rust with exposed modules?
+
+Internally Python maintains the `sys.modules` dict.
+An import in Python is simply a lookup into this dict.
+
+It is necessary to insert the new module into the dict
+before it is exposed to Python. There are two ways of
+doing this.
+
+   * manually inserting into `sys.modules`
+   * using [`append_to_inittab`]({{*PYO3_DOCS_URL}}/pyo3/macro.append_to_inittab.html) macro
+
+
+Example using `sys.modules`
```

Comment:
```suggestion
Example using `PyModule::new` and `sys.modules`:
```

Reply:
Again, too verbose for me.

Reply:
See above. `PyModule::new` is an integral part of this method.

Reply:
I'm happy to repeat `PyModule::new` and `append_to_inittab` again and again. I don't see it as a huge issue.

My preference would still be to put it _once_, and then use less verbose more flowing English after that.